acceptance/tests/rototillerTask/override_command_with_env.rb

```diff
@@ -18,15 +18,15 @@
     rakefile_contents = <<-EOS
       #{rototiller_rakefile_header}
       Rototiller::Task::RototillerTask.define_task :#{@task_name} do |t|
-          t.add_command({:name => 'echo success', :add_env => {:name => '#{override_env}'}})
+          t.add_command({:name => 'echo fail', :add_env => {:name => '#{override_env}'}})
       end
     EOS
     rakefile_path = create_rakefile_on(sut, rakefile_contents)
     sut.add_env_var(override_env, env_value)
 
     execute_task_on(sut, @task_name, rakefile_path) do |result|
       # command was used that was supplied by the override_env
-      assert_match(/^success/, result.stdout, 'The correct command was not observed')
+      assert_match(validation_string, result.stdout, 'The correct command was not observed')
```

Comment:
Don't you have to interpolate validation string into a regex?  I thought this would raise a `TypeError: type mismatch: String given`